src/components/Navigation.res

```diff
@@ -71,7 +71,7 @@ module MobileNav = {
   }
 }
 
-/* isOverlayOpen: if the mobile overlay is toggled open */
+/* isOverlayOpen: if the mobile sidebar is toggled open */
```

Comment:
technically it's not a sidebar on mobile, right?

Reply:
Yeah, good point.